### ISSUE TITLE
OriginInfo example 33

### ISSUE THREAD

**Issue by justinlittman** [sul-dlss-labs/cocina-descriptive-metadata#297]
https://github.com/sul-dlss-labs/cocina-descriptive-metadata/blob/master/mods_cocina_mappings/mods_to_cocina_originInfo.txt#L970

Should it have a roundtrip mapping of
```
<originInfo eventType="publication" lang="rus" script="Cyrl">
  <publisher>СФУ</publisher>
</originInfo>
```
to be consistent with previous example?

**Comment by arcadiafalcone** [comment]
This is fixed in the unmerged PR #287 .

**Comment by arcadiafalcone** [comment]
Closed by #287
